persistence/jdbc/src/test/java/org/infinispan/persistence/jdbc/stringbased/AbstractStringBasedCacheStore.java

```diff
@@ -136,6 +136,10 @@ public EmbeddedCacheManager configureCacheManager(boolean passivation, boolean p
         return defaultCacheManager;
     }
 
+    private void setTableManipulation(JdbcStringBasedStoreConfigurationBuilder storeBuilder) {
```

Comment:
As this method is `private`, I don't understand what value it adds.

Reply:
Good catch, this method should be public, I will update the commit

Reply:
Shouldn't it be `protected` as it's only required by the implementations?

Reply:
@ryanemerson updated